index.html

```diff
@@ -1,32 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Awasome Books</title>
-</head>
+  </head>
+
+  <body>
+    <nav class="navbar">
+      <div>
+        <h3 id="bar-header">Awesome Books</h3>                
+      </div>
+      <div class="nav-list">
+        <a id="list-link" href="#books-header">List</a>
+        <a id="add-new-link" class="add-new" href="#form">Add New</a>
+        <a id="contact-link" href="#contact">Contact</a>
+      </div>
```

Comment:
- [ ] Please when any of the links are clicked, the main area changes to display only one of the three content sections. 
### Please check the project [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) again as you make the correction

Reply:
@CollinsTatang I don't know what you really mean by 'when any of the links are clicked, the main area changes to display only one of the three content sections'
[here](https://jord4n-ac.github.io/Awesome-books/#books-header) is the live link of this project, is it work as the requirements ask
I don't really understand your feedback or what you want me to change

Reply:
### You can check the wire-frame and see how the different nav link display  
![frame](https://user-images.githubusercontent.com/55596970/161347895-e772e48f-b100-4f8b-8ffd-6f5dd572a915.png)

Reply:
@CollinsTatang
I did check the requirements and make sure to meet them.
I'm not sure, but I guess is a deployment problem, When I try it on my browser everything works well, but when I share the link to someone else It doesn't work, I don't really know how to solve this

Reply:
That means you deploy before making recent changes 
Try to deploy the project on notify

Reply:
Push all your changes before deployment

Reply:
- [ ] Please contact your partners to work with you and each and everyone should make multiple commits

Reply:
I deployed after the changes when I finished the project, So I don't really know why this is not working.

Reply:
Thank you, I think that will help.

Reply:
Check the branch you deployed

Reply:
I deployed on the featured branch
I have tried everything but this still doesn't work

Reply:
> Check the branch you deployed
I deployed on the featured branch